src/integer.c

```diff
@@ -1112,114 +1108,115 @@ Obj FuncINT_STRING ( Obj self, Obj string )
 
 /****************************************************************************
 **
-*F  EqInt( <gmpL>, <gmpR> ) . . . . . . . . .  test if two integers are equal
+*F  EqInt( <opL>, <opR> ) . . . . . . . . . .  test if two integers are equal
 **
-**  
-**  'EqInt' returns 1  if  the two integer   arguments <intL> and  <intR> are
-**  equal and 0 otherwise.
+**  'EqInt' returns 1 if the two integer arguments <opL> and <opR> are equal
+**  and 0 otherwise.
 */
-Int EqInt ( Obj gmpL, Obj gmpR )
+Int EqInt(Obj opL, Obj opR)
 {
-  CHECK_INT(gmpL);
-  CHECK_INT(gmpR);
+    CHECK_INT(opL);
+    CHECK_INT(opR);
```

Comment:
Note that here (and also in a few other functions), almost every single line was touched by the argument name change, hence it made sense to simply fully clang-format the function.